test/e2e/gardener/managedseed/create_rotate_delete.go

```diff
@@ -6,246 +6,150 @@ package managedseed
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/ptr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
-	"github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
 	"github.com/gardener/gardener/pkg/apis/seedmanagement/encoding"
 	seedmanagementv1alpha1 "github.com/gardener/gardener/pkg/apis/seedmanagement/v1alpha1"
-	"github.com/gardener/gardener/pkg/client/kubernetes"
 	gardenletconfigv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
-	"github.com/gardener/gardener/pkg/utils/kubernetes/health"
-	. "github.com/gardener/gardener/pkg/utils/test"
-	e2e "github.com/gardener/gardener/test/e2e/gardener"
-	"github.com/gardener/gardener/test/framework"
-	"github.com/gardener/gardener/test/utils/access"
+	. "github.com/gardener/gardener/test/e2e"
+	. "github.com/gardener/gardener/test/e2e/gardener"
+	. "github.com/gardener/gardener/test/e2e/gardener/seed"
+	. "github.com/gardener/gardener/test/e2e/gardener/shoot"
 	"github.com/gardener/gardener/test/utils/rotation"
 )
 
-// GetSeedName returns the name of the managed seed used in this e2e test
-func GetSeedName() string {
-	if os.Getenv("OPERATOR_SEED") == "true" {
-		return "e2e-mngdseed-op"
-	}
-	return "e2e-managedseed"
-}
-
-var parentCtx context.Context
+var _ = Describe("ManagedSeed Tests", Label("ManagedSeed", "default"), Ordered, func() {
+	var s *ManagedSeedContext
```

Comment:
Why is the variable/receiver/param always called `s`?
```suggestion
	var m *ManagedSeedContext
```

Reply:
IMO: I think it's just nice to have the consistency across tests.

We named the test context "s" in the other e2e tests as well:
https://github.com/gardener/gardener/blob/master/test/e2e/gardener/project/project_roles.go
https://github.com/gardener/gardener/blob/master/test/e2e/operator/garden/create_delete.go
https://github.com/gardener/gardener/blob/master/test/e2e/operator/garden/create_rotate_delete.go

Reply:
I was confused to see this in other tests as well. But I don't have a strong opinion.